### PR TITLE
Tags endpoint

### PR DESCRIPTION
**Pull request by Raph22**
Add tags endpoint to retrieve tags

### CODE REVIEW

**Comment by Raph22** [comment]
Test it with:
```js
const Prismic = require('@prismicio/client');

async function main() {
  const client = Prismic.client("https://toto23.cdn.wroom.io/api", {
    accessToken: 'MC5ZSUxmSlJBQUFCc0F3bm9U.77-977-977-9HTbvv73vv71DeTjvv73vv73vv73vv73vv73vv73vv73vv71mUu-_vXLvv73vv71877-9UDDvv73vv70M77-9'
  });

  const api = await client.getApi()
  console.log(JSON.stringify(api))
  const tags = await api.getTags()
  console.log(JSON.stringify(tags))
}

main();
```

**Comment by Raph22** [comment]
Test it with:
```js
const Prismic = require('@prismicio/client');

async function main() {
  const client = Prismic.client("https://toto23.cdn.prismic.io/api", {
    accessToken: 'xxx'
  });

  const api = await client.getApi()
  console.log(JSON.stringify(api))
  const tags = await api.getTags()
  console.log(JSON.stringify(tags))
}

main();
```

**Comment by Raph22** [comment]
Test it with:
```js
const Prismic = require('@prismicio/client');

async function main() {
  const client = Prismic.client("https://<my-repo-name>.cdn.prismic.io/api", {
    accessToken: 'my-access-token'
  });

  const api = await client.getApi()
  console.log(JSON.stringify(api))
  const tags = await api.getTags()
  console.log(JSON.stringify(tags))
}

main();
```

File: src/ResolvedApi.ts [hunk 1]
```diff
@@ -95,6 +95,12 @@ export default class ResolvedApi implements Client {
     return f;
   }
 
+  private tagsForm(): EasyForm {
+    const f = this.form('tags');
+    if (!f) throw new Error('Missing everything form');
```
Comment (lihbr) [inline src/ResolvedApi.ts:100]: Maybe error message should be updated: `Missing tags form` ☺️

**Comment by srenault** [comment]
This one is the new one https://github.com/prismicio/prismic-javascript/pull/170

**Pull request closed without merging**